CHANGELOG.md

```diff
@@ -0,0 +1,498 @@
+# Changelog
+
+## [v5.0.1](https://github.com/substack/tape/releases/tag/v5.0.1)
+Jordan Harband released this on 2020-05-25
+
+ - [Fix] `createStream`: `result` payload is not always an object (#519)
+ - [Fix] Update RegExp for matching stack frames to handle Promise/then scenario (#515)
+ - [readme] add `tape-repeater` (#511)
+ - [Dev Deps] update `eslint`
+
+
+## [v5.0.0](https://github.com/substack/tape/releases/tag/v5.0.0)
+Jordan Harband released this on 2020-04-24
+
+Changes since v4.13.2:
+
+ - [Breaking] only `looseEqual`/`deepEqual, and their inverses, are now non-strict.
+ - [Breaking] make equality functions consistent:
+ - [Breaking] `equal`: use `==`, not `===`, to match `assert.equal`
+ - [Breaking] `strictEqual`: bring `-0`/`0`, and `NaN` into line with `assert`
+ - [Breaking] update `deep-equal` to v2
+ - [Breaking] fail any assertion after `.end()` is called (#489)
+ - [Breaking] `error` should not emit `expected`/`actual` diags (#455)
+ - [Breaking] support passing in an async function for the test callback (#472)
+ - [Breaking] tests with no callback are failed TODO tests (#69)
+ - [Breaking] equality functions: throw when < 2 arguments are provided
+ - [Breaking] add "exports" to restrict public API
+ - [Breaking] `throws`: bring into line with node’s `assert.throws`
+ - [Breaking] use default `require.extensions` collection instead of the magic Array `['.js']` (#396)
+ - [Docs] add an optional emoji version for tap-spec consumer (#501)
+ - [examples] add `ecstatic`
+ - [readme] Add link to tape-player (in-process reporter) (#496)
+ - [meta] change dep semver prefix from ~ to ^
+ - [meta] add `funding` field, create `FUNDING.yml`
+ - [meta] add `auto-changelog`
+ - [Deps] update `deep-equal`, `minimist`, `object-is`, `resolve`
+ - [Dev Deps] update `falafel`
+ - [Tests] Fix simple typo, placehodler -> placeholder (#500)
+
+Changes since v5.0.0-next.5:
+
+ - [Breaking] remove full "lib" export; replace with explicit exports
+ - [examples] add `ecstatic`
+ - [readme] Add link to tape-player (in-process reporter) (#496)
+ - [Docs] add an optional emoji version for tap-spec consumer (#501)
+ - [meta] add `funding` field, create `FUNDING.yml`
+ - [meta] add `auto-changelog`
+ - [Deps] update `deep-equal`, `minimist`, `object-is`, `resolve`
+ - [Dev Deps] update `falafel`
+ - [Tests] Fix simple typo, placehodler -> placeholder (#500)
+
+
+## [v5.0.0-next.5](https://github.com/substack/tape/releases/tag/v5.0.0-next.5)
+Jordan Harband released this on 2020-03-02
```

Comment:
is there a way to link to the github profile, or better npm profile, and be a bit more concise?

Reply:
We can point to an npm release like https://www.npmjs.com/package/tape/v/4.13.0 but might become npm specific. Also GitHub tags doesn't seem to correspond to exact semver versions on npm. Tags like v5.0.0 would be tape/v/5.0.0 on npm.

On a side note I was wondering that GitHub might merge all features of npm into itself and make the release of npm and GitHub into a single thing.

Reply:
I doubt it; npm is intended to be kept separate.

Reply:
Link to npm profile of author? or package?

Concise as in?

Reply:
I'd assume the profile of the author?

By "concise" i mean not a whole sentence; maybe:
```md
## [v5.0.0-next.5](https://github.com/substack/tape/releases/tag/v5.0.0-next.5)
<caption>by [ljharb](https://npmjs.com/~ljharb)</caption>
```
or something (not sure how `caption` renders, maybe `<sub>`)